pages/docs/platform/monitor/insights.mdx

```diff
@@ -0,0 +1,209 @@
+
+### Supported Functions
+
+#### Arithmetic Functions
+Basic mathematical operations and calculations.
+[View ClickHouse arithmetic functions documentation](https://clickhouse.com/docs/sql-reference/functions/arithmetic-functions)
+
+#### String Functions
+String manipulation and search capabilities.
+- [String search functions](https://clickhouse.com/docs/sql-reference/functions/string-search-functions)
+- [String manipulation functions](https://clickhouse.com/docs/sql-reference/functions/string-functions)
+
+#### JSON Functions
+Essential for working with `event_data` payloads.
+[View ClickHouse JSON functions documentation](https://clickhouse.com/docs/sql-reference/functions/json-functions)
+
+#### Date/Time Functions
+For analyzing event timing and patterns.
+[View ClickHouse date/time functions documentation](https://clickhouse.com/docs/sql-reference/functions/date-time-functions)
+
+#### Other Supported Function Categories
+- [Logical functions](https://clickhouse.com/docs/sql-reference/functions/logical-functions)
+- [Rounding functions](https://clickhouse.com/docs/sql-reference/functions/rounding-functions)
+- [Type conversion functions](https://clickhouse.com/docs/sql-reference/functions/type-conversion-functions)
+- [Functions for nulls](https://clickhouse.com/docs/sql-reference/functions/functions-for-nulls)
+- [ULID functions](https://clickhouse.com/docs/sql-reference/functions/ulid-functions)
+
+### Aggregate Functions
+
+The following aggregate functions are supported:
+
+| Function | Description |
+|----------|-------------|
+| `ARRAY_AGG()` | Aggregates values into an array* |
+| `AVG()` | Calculates average |
+| `COUNT()` | Counts rows |
+| `MAX()` | Finds maximum value |
+| `MIN()` | Finds minimum value |
+| `STDDEV_POP()` | Population standard deviation |
+| `STDDEV_SAMP()` | Sample standard deviation |
+| `SUM()` | Calculates sum |
+| `VAR_POP()` | Population variance |
+| `VAR_SAMP()` | Sample variance |
+| `median()` | Finds median value |
+
+> **\*Note on ARRAY_AGG**: Due to a current serialization bug, you need to convert arrays to strings using `toString(ARRAY_AGG(column_name))`.
+
+[View complete ClickHouse aggregate functions documentation](https://clickhouse.com/docs/sql-reference/aggregate-functions/reference)
+
+### SQL Syntax Limitations
+
+Some SQL features are not yet supported but are planned for future releases:
+
+- **CTEs (Common Table Expressions)** using `WITH`
+- **`IS` operator**
+- **`NOT` operator**
+
+
+## Working with Event Data
+
+### Common Schema vs Event-Specific Schema
+
+**Common Schema**: These columns are available for every user and every event:
+- `event_id`
+- `event_name` 
+- `event_ts`
+- `event_v`
+- `event_data`
+
+**Event-Specific Schema**: Within **`event_data`**, users can send any JSON they want, so the structure and available fields will be specific to their payloads. You can use ClickHouse's JSON functions to extract and query specific fields within your event data.
```

Comment:
I think the table you had above already explained the common columns well, so I think this section could just advise to use JSON functions to dig through `data` keys